Tests/Microsoft.Azure.Mobile.Analytics.Test.Windows/SessionTrackerTest.cs

```diff
@@ -55,6 +55,20 @@ public void ResumeAfterTimeout()
             _mockChannel.Verify(channel => channel.EnqueueAsync(It.IsAny<StartSessionLog>()), Times.Exactly(2));
         }
 
+        [TestMethod]
```

Comment:
Could you please add a description for this test like we have for the others? Makes it easier when we need to modify them or debug failures.